CIFAR10/train/basenet.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "bkj",
+    "authorEmail": "ben@canfield.io",
+    "framework": "pytorch",
```

Comment:
What version of PyTorch did you use? Please update with the version number or git commit hash.